scripts/setup_hooks.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+git config core.hooksPath "./hooks"
```

Comment:
OK. I think I want to get a second opinion on this -- @rabidbit @eyurko do you have any thoughts/preferences?